CorsixTH/Lua/world.lua

```diff
@@ -1106,20 +1093,20 @@ function World:onEndDay()
   end
 
   -- check if it's time for an earthquake, and the user is at least on level 5
-  if (self.year - 1) * 12 + self.month == self.next_earthquake.start_month and
-      self.day == self.next_earthquake.start_day then
+  if self.game_date:monthOfGame() == self.next_earthquake.start_month and
```

Comment:
Nicer to use Date for the other side of earthquake and emergency as well.

Reply:
I did it for VIP, but both emergencies and earthquake boundaries are configured within level config, and to translate that might be messy. I would prefer to not mess with it within this PR.